docs/governance/tokenomics.md

```diff
@@ -2,200 +2,178 @@
+In order to succeed, all the generals must attack at the same time. Therefore, they must agree a time to attack collectively. However, communication channels are insecure and messages may be intercepted by traitorous generals. Of **n** generals there are **m** traitors. The challenge is to find an algorithm that ensures a successfuly coordinated attack. 
 
-Validators and Delegators earn block rewards for keeping the blockchain operational and secure.
-Block rewards consist of Staking rewards and Transaction fees.
+Various possibilities are considered and ruled out. In short, Lamport, Shostak, and Pease found that success was only assured when honest generals outnumbered traitors (**m**) by a number greater than **3m + 1** traitors. In other words, coordination or consensus is impossible to achieve if a third or more of the generals are traitors.
 
-## Staking
+:::
 
-The act of staking tokens is called delegating.
+### Incentivisation
 
-A validator needs to self-delegate at least 2M CUDOS.
+Second, incentivisation is built in to the tokenomics model to encourage good behaviour. **Validator operators** who secure the network by running a **Validator node** are required to **self-delegate** a minimum of **2M CUDOS** in return for network rewards. Self-delegating higher amounts increases the chances of that Validator being selected to perform block validation as well as increasing their rewards (as a proportion of the overall amount staked on the network). 
 
-A delegator is someone who delegates any amount of tokens to one or more validators.
+Validators and Delegators earn **Block rewards** for keeping the blockchain operational and secure.
+**Block rewards** consist of **Staking rewards** and **Transaction fees**.
 
-Tokens delegated to a validator, including its self-delegated tokens, are taken out of the delegator’s account and kept in a pool.
+**Tx fees** should be the main source of revenue for **Validators** and **Delegators**. However, due to the expectation of low early engagement numbers on the **Cudos network**, additional rewards are provided to users. 
 
-The validator and its delegators share the rewards based on the amounts they have contributed to this pool.
+:::tip Validator commission
 
-A validator can additionally collect a commission fee from its delegators.
+Validators need to use the `--commission` flag in `cudos-noded CLI` in order to withdraw their commission.
+:::
 
-At start, a validator defines a commission rate, a max change rate, and a max commission rate.
+### Block Rewards 
 
-Then it can change its commission rate once every day by at most its max change rate as long as the commision rate doesn’t become greater than the max commission rate.
+The more tokens delegated to a **Validator node**, the higher the likelihood of it being chosen to propose and sign a block. 
```

Comment:
I we end up removing the following bullet point, we'd have only one sentence here. I suggest let's move this one to incentivization section above, and simply remove the block rewards section.

Reply:
done